packages/block-editor/src/components/inserter/media-tab/media-preview.js

```diff
@@ -203,7 +205,12 @@ export function MediaPreview( { media, onClick, composite, category } ) {
 	const onMouseLeave = useCallback( () => setIsHovered( false ), [] );
 	return (
 		<>
-			<InserterDraggableBlocks isEnabled={ true } blocks={ [ block ] }>
+			<InserterDraggableBlocks
```

Comment:
I think the retrieval of block type info could be part of `BlockDraggable`, like we do for the icon [here](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/block-draggable/index.js#L33).

Reply:
Ok, I'll move it. I placed it here, because the `InserterDraggableBlock` already had the `icon` [parameter](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/inserter-draggable-blocks/index.js#L14) and I thought the expectation was to have it implemented at some point.